src/containers/settings/EditSettingsScreen.tsx

```diff
@@ -860,6 +860,8 @@ class EditSettingsScreen extends Component<EditSettingsScreenProps> {
       this.props.actions.app;
     const form = this.prepareForm();
 
+    console.log('EditSettingsScreen');
```

Comment:
Prior to final merging, I assume we will remove all `console.log` stmts.

Reply:
Correct, there will be no additional console.logs once this PR moves away from draft.